ecms_base/features/custom/ecms_hotel/config/optional/views.view.moderated_content.yml

```diff
@@ -0,0 +1,833 @@
+langcode: en
```

Comment:
Should this configuration be here? Seems like it should be in the workflow module or the profile instead?

Reply:
Ah good catch, removed that.